plugin/src/main/kotlin/net/octyl/ts2kt/gradle/repository/ClientRepository.kt

```diff
@@ -31,7 +31,7 @@ sealed class ResolutionResult {
     data class Success(val files: FileCollection,
                        val dependencies: List<ClientDependency> = listOf()) : ResolutionResult()
 
-    data class NotFound(val error: Throwable? = null) : ResolutionResult()
+    data class Error(val error: Throwable? = null) : ResolutionResult()
```

Comment:
This was used in a few places that make more sense being named 'error'.

Specifically included in the PR because I added another use of the state and thought it was more prudent to reuse the existing sealed state rather than create a third.